Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -0,0 +1,110 @@
+# typed: true
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  SUPPORTED_REPOS = [
+    %w[brew core cask],
+    OFFICIAL_CMD_TAPS.keys.map { |t| t.delete_prefix("homebrew/") },
+    OFFICIAL_CASK_TAPS,
+  ].flatten.freeze
```

Comment:
`cask` is duplicated in this list since it appears in the initial list and also in `OFFICIAL_CASK_TAPS`. I'd suggest using `cask-versions`, `cask-drivers`, and `cask-fonts` as the names for the cask repos to distinguish them from `cask` (i.e. `Homebrew/homebrew-cask`)

Reply:
`OFFICIAL_CASK_TAPS` only shows "cask" and "versions" for me. And "cask" doesn't look like a word any more. :-D

Reply:
I found the problem. `OFFICIAL_CASK_TAPS` had "versions" in it, but the deprecated taps list had "versions" in it too. Turns out, they referred to different repositories: `Homebrew/homebrew-cask-versions` (still alive) and `Homebrew/homebrew-versions` (deprecated). I improved the naming in #13637.